checkstyle/suppressions.xml

```diff
@@ -45,7 +45,7 @@
     <suppress
             checks="ParameterNumber"
             files="(TopicPartitionWriter).java"
-            lines="112,148"
+            lines="149"
```

Comment:
Only one constructor has more than 10 parameters, so the "112" is no longer needed.